src/javascripts/menu items/menuData.js

```diff
@@ -5,10 +5,31 @@ import firebaseConfig from '../helpers/apiKeys';
 
 const dbUrl = firebaseConfig.databaseURL;
 
+// GET MENU ITEMS
 const getMenuItems = () => new Promise((resolve, reject) => {
   axios.get(`${dbUrl}/menu.json`)
     .then((response) => resolve(Object.values(response.data)))
     .catch((error) => reject(error));
 });
 
-export default getMenuItems;
+// DELETE MENU ITEMS
+const deleteMenuItems = (firebasekey, uid) => new Promise((resolve, reject) => {
+  axios.delete(`${dbUrl}/menu/${firebasekey}.json`)
+    .then(() => getMenuItems(uid).then((menuArray) => resolve(menuArray)))
+    .catch((error) => reject(error));
+});
+
+// CREATE MENU ITEMS
```

Comment:
Remove this code. We don't want commented out code in production. This is also considered working ahead!